container/Dockerfile.vllm

```diff
@@ -406,26 +353,51 @@ RUN mkdir -p /opt/dynamo/venv && \
 ENV VIRTUAL_ENV=/opt/dynamo/venv \
     PATH="/opt/dynamo/venv/bin:${PATH}"
 
-ARG ARCH
+ARG TARGETARCH
 # Install vllm - keep this early in Dockerfile to avoid
 # rebuilds from unrelated source code changes
-ARG VLLM_REF
+ARG VLLM_REF="v0.12.0"
 ARG VLLM_GIT_URL
 ARG DEEPGEMM_REF
 ARG FLASHINF_REF
 ARG LMCACHE_REF
-ARG CUDA_VERSION
+ARG CUDA_VERSION="12.9"
 
 ARG MAX_JOBS=16
 ENV MAX_JOBS=$MAX_JOBS
 ENV CUDA_HOME=/usr/local/cuda
 
+# Install sccache if requested
+COPY container/use-sccache.sh /tmp/use-sccache.sh
+# Install sccache if requested
+ARG USE_SCCACHE
+ARG SCCACHE_BUCKET
+ARG SCCACHE_REGION
```

Comment:
Why are we re-enabling sccache here?

In https://github.com/ai-dynamo/dynamo/pull/4866, I disabled sccache for all the frameworks because parallel builds cause socket connection issues on the runner Pod.

I don't see any changes related to resolving this issue in the PR, and its very possible your successful build was lucky to not hit this.

Reply:
The `framework` stages in `main` do not use `sccache` (they don't install or configure it in any way)  https://github.com/ai-dynamo/dynamo/blob/main/container/Dockerfile.vllm

The reason `sccache` was removed was because of the reasoning in my above linked PR. Two stages that build in parallel, (ie. `wheel_builder` and `framework`) could attempt to start two `sccache` servers in parallel, which causes a socket error and fails the build.